### ISSUE TITLE
Question on batch size

### ISSUE THREAD

**Issue by mcreel** [sdatkinson/neural-amp-modeler#212]
In the training code, the batch size is set to 16. There's a youtube video that explores many of the parameters of the net, but not larger batch sizes. In my experience with similar nets, used for non-musical purposes, a larger batch size allows the net to train more effectively. The limitation is that larger batches use up the GPU's memory more quickly. Is GPU memory the reason a batch size of 16 is used? If not, has there been any experimentation with larger batch sizes? I'm running Linux, and the training code doesn't work out of the box (pip can't find a version of onnyxruntime, for the GPU environment), so I haven't been able to play around with it yet.

**Comment by mcreel** [comment]
I have gotten training working with Linux, just by installing python 3.9 instead of 3.11. I did a training run using the standard input file, and the provided overdrive.wav as the target, using a batchsize of 32. Got the same ESR as with batchsize 16. So, at least it's feasible to play with it.

**Comment by sdatkinson** [comment]
The parameters that are the defaults are roughly what should be "good" based on some playing around with them, but I haven't thoroughly checked.

The goal I had in mind when setting the default parameters was to train quickly (i.e. wall time) to a reasonable level of accuracy, not use an overly-heavy model, and be reasonably reliable; _not_ to:
* To use as much of the GPU as possible (though that sometimes coincides with the real goal)
* To get as good of accuracy as possible (ESR=0.01 seems plenty accurate to my ears in most cases.)

But, you should definitely play around with things if you've got other goals or if you've got a reamp that's behaving oddly.
